webapp/src/types/github_types.d.ts

```diff
@@ -0,0 +1,137 @@
+type GithubLabel = {
+    id: number;
+    name: string;
+    color: CSS.Properties;
+}
+
+type User = {
+    login: string;
+}
+
+type Review = {
+    state: string;
+    user: User;
+}
+
+type Item = PrsDetailsData & {
```

Comment:
I would prefer to have these types exported, and explicitly imported elsewhere. Only ones that are imported elsewhere need to be exported.

Also `Item` is a pretty vague name too, so having these global like this is def a bit cryptic. Honestly not sure what `Item` this is referring to. Would `FullPrDetails` be appropriate? Or does this also apply to `Issues` as well?